tests/sceneManager_test.cpp

```diff
@@ -11,6 +11,68 @@ using objects::Object;
 using objects::TriangleObj;
 
 const double kSkipFreq = 1000;
+const core::Vec3 kSkipVec3(0, 0, 0);
 generators::FakeRandomGen fakeRand;
 
-// TODO: Change all doubles to float
+class SceneManagerTest : public ::testing::Test
+{
+protected:
+    SceneManager manager;
+    std::vector<objects::EnergyCollector *> energyCollectors;
+    double energyCollectorRadius = objects::EnergyCollector(kSkipVec3).getRadius();
+    void SetUp() override
+    {
+        energyCollectors = manager.getEnergyCollectors();
+    }
+
+    bool performHitAtEnergyCollectors(const Ray &tempRay, RayHitData *hitData)
+    {
+        bool hit = false;
+        for (const auto &energyColPtr : energyCollectors)
+        {
+            if (energyColPtr->hitObject(tempRay, kSkipFreq, hitData))
+                hit = true;
+        }
+        return hit;
+    }
+
+public:
+    SceneManagerTest(){};
+};
+
+TEST_F(SceneManagerTest, PrintAllEnergyCollectorpositions)
```

Comment:
This test is useless ;-) remove.

Reply:
This was for debuging  to see whats going on only. I will keep this for now, but later when 2nd test will be passed I will delete it.